sbt/src/sbt-test/source-dependencies/pipelining-java/test

```diff
@@ -0,0 +1,8 @@
+# done this way because last modified times often have ~1s resolution
```

Comment:
Would it be possible to define a different read stamps instance that always hashes rather than relying on sleep?

Reply:
Alternatively you could just set the modified times manually of the files after the copy-file. I am nagging about this because CI is already very slow and I'd like to do whatever possible to slow the rate of growth.

Reply:
I think pipelining is somewhat special case situation because it's new and timing needs to be tested for both Linux and Windows.

Reply:
I don't understand where and why timing matters. My assumption is that timestamps matter because the second time you run compile after a `copy-file`, it isn't guaranteed to run due to file hash caching using timestamps. Is that not the reason for the sleep?

Regardless, it seems to me that we should have a setting to disable timestamp caching since it could break incremental compilation for file systems that don't even support setting the last modified (I think that's a thing).

Reply:
It's also possible that I had misattributed some during-development failure to timestamp issue, and sleep is no longer needed. It would be good to probably confirm and smoke out exactly what invalidation gets affected.

Reply:
Removed sleep in 002f97c.